docs/changelog.md

```diff
@@ -5,12 +5,22 @@ toc:
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.130 (2023-07-?)
+
+### Features
+
+- Added new rule `component-name-unique` to check for unique component names
```

Comment:
Do we thank external contributors? Can we?

Reply:
```suggestion
- Added new rule `component-name-unique` to check for unique component names (kudos to @pboos 🎉).
```